lib/plivo/resources/brand.rb

```diff
@@ -54,7 +54,134 @@ def list(options=nil)
   
         ##
         # Create a new brand
-      def create(params)
+      def create(options=nil)
+        valid_param?(:options, options, Hash, true)
+        params = {}
+        if options.key?(:brand_alias) 
+          params[:brand_alias] = options[:brand_alias]
+        else
+          raise_invalid_request("brand_alias must be provided")
+        end
+        if options.key?(:brand_type) 
+          params[:brand_type] = options[:brand_type]
+        else
+          raise_invalid_request("brand_type must be provided")
+        end
+        if options.key?(:profile_uuid) 
+          params[:profile_uuid] = options[:profile_uuid]
+        else
+          raise_invalid_request("profile_uuid must be provided")
+        end
+        if options.key?(:brand_type) 
+          params[:brand_type] = options[:brand_type]
+        else
+          raise_invalid_request("brand_type must be provided")
+        end
+        if options.key?(:secondary_vetting) 
+          params[:secondary_vetting] = options[:secondary_vetting]
+        else
```

Comment:
if we not pass, automatically it will be false. Please remove the else condition